config.json

```diff
@@ -6,6 +6,8 @@
     "period" : "period",
     "strata" : "group",
     "reference" : "reference",
+    "auxiliary": "auxiliary",
+    "predictive_auxiliary": "f_match_auxiliary",
```

Comment:
` "predictive_auxiliary": "f_match_auxiliary"` isn't this intermediate column ?  Why the user needs control of it?

Reply:
Removed from Config